### PR TITLE
fpco-40299-devtools

### PR DESCRIPTION
**Pull request by girishbharambe**

### CODE REVIEW

File: src/main/resources/application-prod.yml [hunk 1]
```diff
@@ -0,0 +1,11 @@
+# FDK Extension Configuration (same as in the default profile)
+fdk-extension:
+  version: '0.5.0'  # Version of the FDK extension being used
+
+# Spring Boot Developer Tools Configuration (for production environment)
+spring:
```
Comment (vivek-gofynd) [inline src/main/resources/application-prod.yml:17]: Is this need to define explicitly for production ?

**Review by vivek-gofynd — CHANGES_REQUESTED** [review]

File: Dockerfile [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+WORKDIR /app
+COPY ./pom.xml ./
+COPY ./src ./src
+RUN apt-get update && apt-get install -y maven && mvn dependency:go-offline -B
+COPY --from=frontend-builder /app/build ./src/main/resources/static/
+RUN mvn clean package -DskipTests
+FROM openjdk:17-slim
+WORKDIR /app
+COPY --from=backend-builder /app/target/example-extension-java-react-0.0.1-SNAPSHOT.jar ./example-extension-java-react-0.0.1-SNAPSHOT.jar
+COPY src/main/resources/application.yml ./application.yml
```
Comment (vivek-gofynd) [inline Dockerfile]: Is this required for production ? @girishbharambe

**Review by vivek-gofynd — CHANGES_REQUESTED** [review]

File: Dockerfile (hunk 2 above)
Comment (girishbharambe) [reply Dockerfile]: If user wants to run application locally on docker, application.yml will need to be moved. I have given instructions to set env in readme

File: src/main/resources/application-prod.yml (hunk 1 above)
Comment (girishbharambe) [reply src/main/resources/application-prod.yml:17]: Anyways when user will make the extension live, he ll need to create this separate yml

**Review by vivek-gofynd — APPROVED** [review]

**Pull request merged**